backend/common/utils/cxg_generation_utils.py

```diff
@@ -31,6 +34,29 @@ def convert_dictionary_to_cxg_group(cxg_container, metadata_dict, group_metadata
             metadata_array.meta[key] = value
 
 
+def convert_uns_to_cxg_group(cxg_container, metadata_dict, group_metadata_name="uns", ctx=None):
+    """
+    Convert uns (unstructured) metadata to CXG output directory specified
+    Generate deep zoom assets for spatial data
+    """
+
+    spatial_processor = SpatialDataProcessor()
+
+    array_name = f"{cxg_container}/{group_metadata_name}"
+    object_filtered = {}
+
+    tiledb.from_numpy(array_name, np.zeros((1,)))
+
+    with tiledb.open(array_name, mode="w", ctx=ctx) as metadata_array:
```

Comment:
@tihuan I believe the costly part mentioned is opening the h5ad, since were already doing it as part of the cxg conversion, we're just piggy backing/adding a step here for the `uns` conversion and generating deep zoom assets - @atarashansky